mcstatus/forge_data.py

```diff
@@ -0,0 +1,178 @@
+"""Forge Data Decoder
+
+Forge mod data is encoded into a UTF-16 string that represents
+binary data containing data like the forge mod loader network
+version, a big list of channels that all the forge mods use,
+and a list of mods the server has.
+
```

Comment:
```suggestion

Before it was in `forgeData` attribute inside a response object. We support this implementation too.

```
(String length may be incorrect in this suggestion)